debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/SnapshotReader.java

```diff
@@ -541,7 +546,8 @@ protected void execute() {
                 // ------
                 // STEP 7
                 // ------
-                if (snapshotLockingMode.equals(MySqlConnectorConfig.SnapshotLockingMode.MINIMAL) && isLocked) {
+                if ((snapshotLockingMode.equals(MySqlConnectorConfig.SnapshotLockingMode.MINIMAL) ||
```

Comment:
Could this be retrieved from a method on the enum, e.g. `isMinimal()`, or `requiresMinimalLocking()` or something similar?

Reply:
I went with `usesMinimalLocking` -- the read best to me, but happy with any name, the method is a good suggestion 👍